_sass/base/_highlight.scss

```diff
@@ -102,15 +102,15 @@ table.rouge-table {
   .bp{ /* Name.Builtin.Pseudo */
     color: $base0c  }
   .c { /* Comment */
-    color: $base04 }
+    color: $base03 }
   .cm{ /* Comment.Multiline */
-    color: $base04  }
+    color: $base03  }
   .cp{ /* Comment.Preproc */
-    color: $base04  }
+    color: $base03  }
   .c1{ /* Comment.Single */
-    color: $base04  }
+    color: $base03  }
   .cs{ /* Comment.Special */
-    color: $base04  }
+    color: $base03  }
```

Comment:
I don't think the problem is with the `$base03` comments should be in `$base04`. It does look darker on your screenshot than for me which is not normal.

Here with `base04`:
![image](https://user-images.githubusercontent.com/20642750/112672266-abb55580-8e39-11eb-8255-07d72415f746.png)

And here is too dark with `base03`:
![image](https://user-images.githubusercontent.com/20642750/112672359-c5ef3380-8e39-11eb-876e-ef4ce13e6c2d.png)

Could it be the `base16` theme rendering differently?

Reply:
I wrote comments in the image description and forgot they would not render.

First image have right comment color but wrong selection.
Second image have wrong comment color but right selection.

The darker one for the comment (base03) is the right one.

I used the same base16 theme.

Reply:
Alright, I see what you mean, then I think that the theme was mixed up and put a darker one for the comments.
I'll refer to http://chriskempson.com/projects/base16/